nautobot/docs/installation/http-server.md

```diff
@@ -0,0 +1,199 @@
+
+Two files will be created: the public certificate (`nautobot.crt`) and the private key (`nautobot.key`). The certificate is published to the world, whereas the private key must be kept secret at all times.
+
+!!! info
+    Some Linux installations have changed the location for SSL certificates from `/etc/ssl/` to `/etc/pki/tli/`. The
+    command below may need to be changed to reflect the certificate location.
+
+    The following command will prompt you for additional details of the certificate; all of which are optional.
+
+```no-highlight
+$ sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 \
+  -keyout /etc/ssl/private/nautobot.key \
+  -out /etc/ssl/certs/nautobot.crt
+```
+
+## HTTP Server Installation
+
+Any HTTP server of your choosing is supported. For your convenience, setup guides for the most common options are
+provided here.
+
+### NGINX
+
+[NGINX](https://www.nginx.com/resources/wiki/) is a free, open source, high-performance HTTP server and reverse proxy
+and is by far the most popular choice.
+
+#### Install NGINX
+
+Begin by installing NGINX:
+
+```no-highlight
+$ sudo apt install -y nginx
+```
+
+#### Configure NGINX
+
+!!! warning
+    The following steps must be performed with root permissions.
+
+Once NGINX is installed, copy and paste the following NGINX configuration into
+`/etc/nginx/sites-available/nautobot.conf`: 
+
+```
+server {
+    listen 443 ssl;
+
+    # CHANGE THIS TO YOUR SERVER'S NAME
+    server_name nautobot.example.com;
+
+    ssl_certificate /etc/ssl/certs/nautobot.crt;
+    ssl_certificate_key /etc/ssl/private/nautobot.key;
+
+    client_max_body_size 25m;
+
+    location /static/ {
+        alias /opt/nautobot/static/;
+    }
+
+    location / {
+        proxy_pass http://127.0.0.1:8001;
+        proxy_set_header X-Forwarded-Host $http_host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-Proto $scheme;
+    }
+}
+
+server {
+    # Redirect HTTP traffic to HTTPS
+    listen 80;
+    server_name _;
+    return 301 https://$host$request_uri;
+}
+```
+
+- Be sure to replace `nautobot.example.com` with the domain name or IP address of your installation. This should match
+the value configured for `ALLOWED_HOSTS` in `nautobot_config.py`.
+- If the file location of SSL certificates had to be changed in the [Obtain an SSL
+  Certificate](#obtain-an-ssl-certificate) step above, then the location will need to be changed in the NGINX
+  configuration you pasted.
+
+#### For CentOS or RHEL users
+
+Red Hat-based (namely CentOS & RHEL) systems running NGINX will need to create the directory structure and perform a
+minor update the default `nginx.conf` file to get it to read included configurations.
+
+##### Create the include directories
+To do this, create the `sites-available` and `sites-enabled` directories.
+
+```no-highlight
+$ sudo mkdir -p /etc/nginx/{sites-available,sites-enabled}
```

Comment:
Still feel like this belongs above the `Configure NGINX` section.

Reply:
Agreed.